packages/apps/spaces/styles/globals.scss

```diff
@@ -95,3 +95,12 @@
 .lp_pass {
   display: none !important;
 }
+@keyframes line-animated {
```

Comment:
I would do this with chakra in `@ui/util/keyframes` and import it from there.

Reply:
I will remove that altogether in the next pr due to pivoting from line created in pseudoelement to text decoreation effect